src/mainFull.js

```diff
@@ -0,0 +1,1409 @@
+  createMagneticConstant,
+  createMagneticFluxQuantum,
+  createMolarMass,
+  createMolarMassC12,
+  createMolarPlanckConstant,
+  createMolarVolume,
+  createNeutronMass,
+  createNuclearMagneton,
+  createPlanckCharge,
+  createPlanckConstant,
+  createPlanckLength,
+  createPlanckMass,
+  createPlanckTemperature,
+  createPlanckTime,
+  createProtonMass,
+  createQuantumOfCirculation,
+  createReducedPlanckConstant,
+  createRydberg,
+  createSackurTetrode,
+  createSecondRadiation,
+  createSpeedOfLight,
+  createStefanBoltzmann,
+  createThomsonCrossSection,
+  createVacuumImpedance,
+  createWeakMixingAngle,
+  createWienDisplacement
+} from './type/unit/physicalConstants'
+import { createFilterTransform } from './expression/transform/filter.transform'
+import { createForEachTransform } from './expression/transform/forEach.transform'
+import { createIndexTransform } from './expression/transform/index.transform'
+import { createMapTransform } from './expression/transform/map.transform'
+import { createMaxTransform } from './expression/transform/max.transform'
+import { createMeanTransform } from './expression/transform/mean.transform'
+import { createMinTransform } from './expression/transform/min.transform'
+import { createRangeTransform } from './expression/transform/range.transform'
+import { createSubsetTransform } from './expression/transform/subset.transform'
+import { createConcatTransform } from './expression/transform/concat.transform'
+import { createSumTransform } from './expression/transform/sum.transform'
+import { MATRIX_OPTIONS, NUMBER_OPTIONS } from './core/function/config'
+import { createApply } from './function/matrix/apply'
+import { createColumn } from './function/matrix/column'
+import { createRow } from './function/matrix/row'
+import { createApplyTransform } from './expression/transform/apply.transform'
+import { createStdTransform } from './expression/transform/std.transform'
+import { createVarianceTransform } from './expression/transform/variance.transform'
+import { createRowTransform } from './expression/transform/row.transform'
+import { createColumnTransform } from './expression/transform/column.transform'
+import { createDeprecatedEval } from './expression/function/eval'
+import { createDeprecatedImport } from './core/function/deprecatedImport'
+import {
+  isAccessorNode,
+  isArrayNode,
+  isAssignmentNode, isBigNumber,
+  isBlockNode, isBoolean,
+  isChain, isComplex,
+  isConditionalNode,
+  isConstantNode,
+  isDate, isDenseMatrix, isFraction, isFunction,
+  isFunctionAssignmentNode,
+  isFunctionNode, isHelp, isIndex,
+  isIndexNode, isMatrix,
+  isNode,
+  isNull,
+  isNumber,
+  isObject,
+  isObjectNode,
+  isOperatorNode,
+  isParenthesisNode, isRange,
+  isRangeNode,
+  isRegExp, isResultSet, isSparseMatrix,
+  isSymbolNode, isUnit
+} from './utils/is'
+import { isUndefined } from 'karma/common/util'
+import { isArray } from 'karma/lib/helper'
+import { isString } from 'mocha/lib/utils'
```

Comment:
It is because we are importing these scripts that browser tests are failing.

These scripts are designed to work on node (and probably not meant to be used outside of karma) we should writing these functions our selves or maybe using lodash for them.